openapi/src/parts/controlled_azure_database.yaml

```diff
@@ -72,6 +72,50 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/ServerError'
+    post:
+      summary: |
+        Delete a controlled Azure Database resource. This is async, because the 
+        operation can take a few minutes. OpenAPI does not support request body 
+        in DELETE, but async state requires it. Hence this is a POST.
+      operationId: deleteAzureDatabaseAsync
+      tags: [ ControlledAzureResource ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeleteControlledAzureResourceRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/DeleteControlledAzureResourceResponse'
```

Comment:
hmm... is this a valid response for a POST?

For disk I think I had to use `CreateControlledAzureResourceResult` over `CreateControlledAzureResourceResponse` for the `post`, and then `...Response` for the subsequent job `get`
```
responses:
        '200':
          description: Response to create controlled Azure disk
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/CreateControlledAzureResourceResult'
```

Reply:
🤔 I am following the example of the deleteDisk async POST that is using `Response` so I do not think making a `Result` is needed. @aherbst-broad is there an established best practices that you would like us to follow for WSM?

Reply:
yeah I think create is different than delete in that the create result returns more information about the resource itself than the response on delete (when it isn't necessary to have the resourceId)

Reply:
TLDR; using the response should be fine